lib/eclipse/EclipseState/Schedule/WellInjectionProperties.cpp

```diff
@@ -27,6 +27,7 @@ namespace Opm {
     WellInjectionProperties::WellInjectionProperties() {
         surfaceInjectionRate=0.0;
         reservoirInjectionRate=0.0;
+        temperature=273.15 + 15.56;
```

Comment:
Please avoid the use of magic numbers - the `273.15`can be found [here](https://github.com/OPM/opm-parser/blob/f763c862a6e1efc8b2d0d1e5d047f78aae31c9e8/lib/eclipse/include/opm/parser/eclipse/Units/Units.hpp#L161) - can add another named constant for `15.56`.

Reply:
Ok, changed&pushed. for the 15.56 degrees Celsius, I misused the default value of the STCOND keyword...